pages/common/docker.md

```diff
@@ -4,36 +4,32 @@
 
 - List currently running docker containers:
 
-`docker container ls`
+`docker ps`
 
 - List all docker containers (running and stopped):
 
-`docker container ls -a`
+`docker ps -a`
 
 - Start a container from an image, with a custom name:
 
-`docker container run --name={{container_name}} {{image}}`
+`docker run --name {{container_name}} {{image}}`
 
 - Start or stop an existing container:
 
-`docker container {{start|stop}} {{container_name}}`
+`docker {{start|stop}} {{container_name}}`
 
 - Pull an image from a docker registry:
 
 `docker pull {{image}}`
 
-- Start a container from an image and get a shell inside of it:
-
-`docker container run -it {{image}} bash`
-
 - Run a command inside of an already running container:
 
-`docker container exec {{container_name}} {{command}}`
+`docker exec -it {{container_name}} {{command}}`
```

Comment:
I think it will be better if we change command to bash. That might preserve the information we already had in the page.

Reply:
Changed it to `sh` as that's more guaranteed to exist on a container than bash ([Alpine Linux](https://alpinelinux.org/) is a popular container image that doesn't have bash for example).

Reply:
Sure